CODEOWNERS

```diff
@@ -349,6 +349,7 @@ homeassistant/components/raincloud/* @vanstinator
 homeassistant/components/rainforest_eagle/* @gtdiehl @jcalbert
 homeassistant/components/rainmachine/* @bachya
 homeassistant/components/random/* @fabaff
+homeassistant/components/rejseplanen/* @DarkFox
```

Comment:
Please update the rejseplanen manifest file too.

Reply:
D'oh! I knew I'd missed something. 👍